### PR TITLE
Travis-CI xcode11.2

### PR DESCRIPTION
**Pull request by kevnm67**
## Pull request checklist

- [x] All tests pass. 
- [x] Demo project builds and runs.
- [x] I have resolved merge conflicts.
- [x] I have followed the [coding style](https://github.com/jessesquires/HowToContribute#style-guidelines) and [contributing guidelines](https://github.com/jessesquires/HowToContribute).

#### This fixes issue #
- Broken Travis-CI builds.

## What's in this pull request?
- Bump osx image to xcode11.2
- Updated iphone and tv simulator SDK's.
- Bump ruby to 2.6.4 - comes pre-installed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,47 +21,50 @@ env:
...
+  - make setup
 
 jobs:
   include:
     - &test
       stage: unit tests
-      name: iOS 11.1
-      env: DESTINATION="OS=11.1,name=iPhone X" SDK="$IOS_SDK"
+      name: iOS 11.4
+      env: DESTINATION="OS=11.4,name=iPhone X" SDK="$IOS_SDK"
```
Comment (jessesquires) [inline .travis.yml]: can we leave this at `11.1` or is that no longer available?

File: .travis.yml [hunk 2]
```diff
@@ -21,47 +21,50 @@ env:
...
         - xcodebuild clean test -project "$PROJECT" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" -enableCodeCoverage YES -configuration Debug ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO | bundle exec xcpretty -c;
       after_success:
-        - bundle exec danger
+        - make danger
         - bash <(curl -s https://codecov.io/bash)
 
     - <<: *test
-      name: iOS 12.0
-      env: DESTINATION="OS=12.0,name=iPhone X" SDK="$IOS_SDK"
+      name: iOS 12.1
```
Comment (jessesquires) [inline .travis.yml]: can we leave at `12.0` ?

File: .travis.yml [hunk 3]
```diff
@@ -21,47 +21,50 @@ env:
...
+      env: DESTINATION="OS=12.1,name=iPhone X" SDK="$IOS_SDK"
+
+    - <<: *test
+      name: iOS 13.2
```
Comment (jessesquires) [inline .travis.yml:51]: can this stay `13.0` ?

File: .travis.yml [hunk 4]
```diff
@@ -21,47 +21,50 @@ env:
...
+      env: DESTINATION="OS=11.0,name=Apple TV" SDK="$TVOS_SDK"
 
     - <<: *test
       name: tvOS 12.0
       env: DESTINATION="OS=12.0,name=Apple TV" SDK="$TVOS_SDK"
 
     - <<: *test
-      name: tvOS 13.0
-      env: DESTINATION="OS=13.0,name=Apple TV" SDK="$TVOS_SDK"
+      name: tvOS 13.2
```
Comment (jessesquires) [inline .travis.yml:63]: can this stay `13.0` ?

File: .travis.yml [hunk 5]
```diff
@@ -81,8 +84,8 @@ jobs:
       env: DESTINATION="OS=12.0,name=iPhone X" SDK="$IOS_SDK"
 
     - <<: *build-examples
-      name: iOS 13.0
-      env: DESTINATION="OS=13.0,name=iPhone 11" SDK="$IOS_SDK"
+      name: iOS 13.2
```
Comment (jessesquires) [inline .travis.yml:88]: can this stay `13.0` ?

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+endif
+
+install_gems:
+	$(info )
+	$(info Install Ruby Gems ...)
+
+	bundle install --without=documentation
+
+danger:
+	bundle exec danger
```
Comment (jessesquires) [inline Makefile]: this is awesome :smile: but feels a little to heavy compared to just having a `.travis.yml`

could we just update the travis config to use `gem update bundler` and `bundle install --without=documentation` ?

**Review by jessesquires — CHANGES_REQUESTED** [review]
thanks @kevnm67 ! 💯 👏 

a few minor comments

RE: iOS versions. I prefer to use the lowest available major version (so 11.1, 12.0, 13.0). 
Curious that 11.0 isn't available 🤷‍♂

File: .travis.yml (hunk 1 above)
Comment (kevnm67) [reply .travis.yml]: It should be.

Sorry about the versioning.  I was looking at a previous, passing revision in a first attempt to solve the failure.  Will keep your preference in mind moving forward 😉

File: Makefile (hunk 6 above)
Comment (kevnm67) [reply Makefile]: Ha, sure.  

I was expecting that 😄.  My original thought/intention was extracting the script stage logic but felt it was out of scope.  Also, wasn't sure your thoughts on that approach.  It's something I did at work to share common commands between multiple projects.

**Review by jessesquires — APPROVED** [review]

**Comment by jessesquires** [comment]
thanks @kevnm67 ! 💯

**Pull request merged**